strings/src/main/java/com/revenuecat/purchases/strings/PurchaseStrings.kt

```diff
@@ -25,4 +25,5 @@ object PurchaseStrings {
     const val UPGRADING_SKU = "Moving from old SKU %s to sku %s"
     const val UPGRADING_SKU_ERROR = "There was an error trying to upgrade. BillingResponseCode: %s"
     const val NOT_RECOGNIZED_PRODUCT_TYPE = "Type of product not recognized."
+    const val SKIPPING_AUTOMATIC_SYNC = "Skipping automatic synchronization."
```

Comment:
Can we add something more description to the user if this is set? Like...

```
Skipping automatic synchronization. Call `syncPurchases` when you are ready.
```

But probably better wording? 🙃 There is a chance this might end up in less but also not 🤷‍♂️